### DISCUSSION TITLE
Kitty automatic exit condition.

### DISCUSSION THREAD

**Discussion by 2403772980ygy** [kovidgoyal/kitty#6789] [Q&A] [answered]
Whenever I start a background process and disown it, then run exit in the shell, kitty does not close.

How can I make kitty close on such case?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.close_on_child_death

**Reply by 2403772980ygy** [reply]
That was quick! Thanks for the quick reply and answer.
